tests/modules/movie_test.cpp

```diff
@@ -7,57 +7,94 @@
 #include "movie_data.h"
 
 using namespace ::testing;
+using namespace faker;
 using namespace faker::movie;
 
-class MovieTest : public Test
+namespace
+{
+const struct MovieDefinition& getMovie(Locale locale)
+{
+    switch (locale)
+    {
+    default:
+        return enUSmoviesDefinitions;
+    }
+}
+}
+
+class MovieTest : public TestWithParam<Locale>
 {
 public:
 };
 
-TEST_F(MovieTest, shouldGenerateGenre)
+TEST_F(MovieTest, shouldGenerateGenreLocale)
```

Comment:
TEST_P is needed when tests with param

Reply:
I have fixed it in the latest commit, can you please review it.